contract/ignition/modules/OfficerModule.ts

```diff
@@ -3,9 +3,14 @@ import boardOfDirectorsBeaconModule from './BoardOfDirectorsBeaconModule'
 import VotingBeaconModule from './VotingBeaconModule'
 
 export default buildModule('Officer', (m) => {
+  const beaconAdmin = m.getAccount(0)
   const officer = m.contract('Officer')
-  const { beacon } = m.useModule(boardOfDirectorsBeaconModule)
+  const { beacon: BoDBeacon } = m.useModule(boardOfDirectorsBeaconModule)
   const { beacon: votingBeacon } = m.useModule(VotingBeaconModule)
-  m.call(officer, 'initialize', [beacon, votingBeacon])
-  return { officer }
+  m.call(officer, 'initialize', [BoDBeacon, votingBeacon])
```

Comment:
no need to initialize, initialize will happen on frontend deployment as a user

Reply:
Got it.